lib/ids_please.rb

```diff
@@ -73,5 +72,4 @@ def recognize_link(link)
     end
     unrecognized << link
   end
-
```

Comment:
I like those gaps in beginning and end of class/module =)
I don't do this gap in beginning only if there some `import` or `extend` as it tells the reader that class or module is extended with methods from somewhere or it's not a usual class. Style preferences ;)